functions/src/definitions/eventHandlers/onVoteRequestUpdate.ts

```diff
@@ -19,29 +20,31 @@ if (!admin.apps.length) {
 
 const db = admin.firestore()
 
-const onVoteRequestUpdate = functions
-  .region("asia-southeast1")
-  .runWith({
+const onVoteRequestUpdateV2 = onDocumentUpdated(
+  {
+    document: "messages/{messageId}/voteRequests/{voteRequestId}",
     secrets: ["WHATSAPP_CHECKERS_BOT_PHONE_NUMBER_ID", "WHATSAPP_TOKEN"],
-  })
-  .firestore.document("/messages/{messageId}/voteRequests/{voteRequestId}")
-  .onUpdate(async (change, context) => {
+  },
+  async (event) => {
     // Grab the current value of what was written to Firestore.
-    const before = change.before.data()
-    const docSnap = change.after
-    const after = docSnap.data()
+    if (!event?.data?.before || !event?.data?.before) {
+      return Promise.resolve()
+    }
+    const before = event.data.before.data()
```

Comment:
I think a more descriptive name for `before` and `after` would be beneficial

Reply:
Ok changed!